# Appendix files for Ross Laidlaw's PhD thesis 

The following repository contains all the appendix files for the PhD thesis: Applications of scRNA-seq to capture Trypanosomatid life cycle stage transitions and heterogeneity.

#Chapter 1 appendix files

Appendix file 1

TrAGEDy DE results across four windows of comparison for the TrAGEDy aligned WT vs ZC3H20 KO _T. brucei_ trajectories. Significant DE genes are identified as those with an absolute Log<sub>2</sub>FC of 0.5, bonferroni adjusted p-value less than 0.05 and were expressed in at least 10\% of cells in one of the conditions in the window.

Appendix file 2

TradeSeq DE results across four knots of comparisons for Seurat integrated WT vs ZC3H20 KO _T. brucei_ trajectories. Significant DE genes are identified as those with an absolute Log<sub>2</sub>FC of 0.5, bonferroni adjusted p-value less than 0.05 and were expressed in at least 10\% of cells in one of the conditions in the knots.

Appendix file 3

Seurat DE results across four clusters of comparisons for Seurat integrated WT vs ZC3H20 KO _T. brucei_ trajectories. Significant DE genes are identified as those with an absolute Log<sub>2</sub>FC of 0.5, bonferroni adjusted p-value less than 0.05 and were expressed in at least 10\% of cells in one of the conditions in the clusters.

Appendix file 4

Table of the runtimes (in minutes) taken by TrAGEDy, Seurat and TradeSeq to return DE results for the _T. brucei_ and T cell datasets

Appendix file 5

Significant (Benjamini p-value less than 0.05) GO terms, generated using TriTrypDB, for all the significant DE genes captured by TrAGEDy, tradeSeq and Seurat for the WT vs ZC3H20 KO _T. brucei_ comparison.

Appendix file 6

Significant (Benjamini p-value less than 0.05) GO terms, generated using TriTrypDB, for all the uniquely captured DE genes by TrAGEDy, tradeSeq and Seurat for the WT vs ZC3H20 KO _T. brucei_ comparison.

Appendix file 7

Genes IDs and associated Log<sub>2</sub>FC for the heatmap in figure [insert*]

Appendix file 8

TrAGEDy DE results across four windows of comparison for the TrAGEDy aligned WT vs Bcl11b KO T cell trajectories. Significant DE genes are identified as those with an absolute Log<sub>2</sub>FC of 0.5, bonferroni adjusted p-value less than 0.05 and were expressed in at least 10\% of cells in one of the conditions in the window.

Appendix file 9

TradeSeq DE results across four knots of comparisons for Seurat integrated WT vs Bcl11b KO T cell trajectories. Significant DE genes are identified as those with an absolute Log<sub>2</sub>FC of 0.5, bonferroni adjusted p-value less than 0.05 and were expressed in at least 10\% of cells in one of the conditions in the knots.

Appendix file 10

Seurat DE results across four clusters of comparisons for Seurat integrated WT vs Bcl11b KO T cell trajectories. Significant DE genes are identified as those with an absolute Log<sub>2</sub>FC of 0.5, bonferroni adjusted p-value less than 0.05 and were expressed in at least 10\% of cells in one of the conditions in the clusters.

Appendix file 11

Significant (Benjamini p-value less than 0.05) GO terms, generated using clusterProfiler, for all the significant DE genes captured by TrAGEDy, tradeSeq and Seurat for the WT vs Bcl11b KO T cell comparison.

Appendix file 12

Significant (Benjamini p-value less than 0.05) GO terms, generated using clusterProfiler, for all the uniquely captured DE genes by TrAGEDy, tradeSeq and Seurat for the WT vs Bcl11b KO T cell comparison.

Appendix file 13

Genes IDs and associated Log<sub>2<sub>FC for the heatmap in figure [insert*]


#Chapter 2 appendix files

Appendix file 14

The significant marker genes of the clusters in the _T. cruzi_ atlas. Significant marker genes were defined as the genes which had a bonferroni adjusted p-value < 0.05 and a Log<sub>2</sub>FC > 0.5. 

Appendix file 15

The significant marker genes of the bulk RNA-seq _T. cruzi_ lifecycle stage samples, with the amastigotes merged together into one sample. Significant marker genes were defined as the genes which had a bonferroni adjusted p-value < 0.05 and a Log<sub>2</sub>FC > 0.

Appendix file 16

The significant marker genes of the bulk RNA-seq _T. cruzi_ lifecycle stage samples, with the amastigotes separated by hours post infection. Significant marker genes were defined as the genes which had a bonferroni adjusted p-value < 0.05 and a Log<sub>2</sub>FC > 0.

Appendix file 17

List of genes which are significantly DE between the trypomast_0 and trypomast_4 clusters. Significant DE genes were defined as the genes which had a bonferroni adjusted p-value < 0.05 and an absolute Log<sub>2</sub>FC > 0.5.

Appendix file 18 

List of genes which are significantly associated with the metacyclogenesis process. Significant marker genes were defined as the genes which had a bonferroni adjusted p-value < 0.05 and a Log<sub>2</sub>2FC > 0.5 and were expressed in at least 10% of cells in the metacyclogenesis process.


#Chapter 3 appendix files

Appendix file 19 

Alignment of the JM042 Sanger sequence of the RBP6 ORF ligated into the pLEW100v5 plasmid against the Lister 427 RBP6 ORF.

Appendix file 20 

Alignment of the HDK430 Sanger sequence of the RBP6 ORF ligated into the pLEW100v5 plasmid against the Lister 427 RBP6 ORF.

Appendix file 21 

Table showing the significant marker genes (bonferroni adjusted p-value < 0.05, Log<sub>2</sub>FC > 0.5 and were expressed in at least 10\% of cells in the cluster being compared) for each of the 7 clusters shown in Figure [*insert].

Appendix file 22 

Table showing the significant DE genes (bonferroni adjusted p-value < 0.05, Log<sub>2</sub>FC > 0.5 and were expressed in at least 10\% of cells in the cluster being compared) between the RBP6 overexpression generated metacyclics and _in vivo_ derived metacyclics, taken from Hutchinson and colleagues (Hutchinson et al., 2021).

Appendix file 23 

Table showing the enrichment of metabolic processes (as derived from KEGG and Metacyc) in the DE genes which were higher expressed in the _in vivo_ metacyclic cells compared with RBP6 overexpression derived metacyclic cells.

Appendix file 24 

Table showing the enrichment of metabolic processes (as derived from KEGG and Metacyc) in the DE genes which were higher expressed in the RBP6 overexpression derived metacyclic cells compared with _in vivo_ metacyclic cells.

Appendix file 25 

Table showing the significant DE genes (bonferroni adjusted p-value < 0.05, Log<sub>2</sub>FC > 0.5 and were expressed in at least 10\% of cells in the cluster being compared) between the RBP6 overexpression generated metacyclics and _in vitro_ cultured PCFs, taken from Briggs and colleagues (Briggs et al., 2023).

Appendix file 26 

Table showing the enrichment of metabolic processes (as derived from KEGG and Metacyc) in the DE genes which were higher expressed in the _in vitro_ PCFs compared with RBP6 overexpression derived metacyclic cells.

Appendix file 27 

Table showing the enrichment of metabolic processes (as derived from KEGG and Metacyc) in the DE genes which were higher expressed in the RBP6 overexpression derived metacyclic cells compared with _in vitro_ PCFs.

Appendix file 28 

Table showing the significant DE genes (bonferroni adjusted p-value < 0.05, Log<sub>2</sub>FC > 0.5 and were expressed in at least 10\% of cells in the cluster being compared) between the early PCFs contained within the RBP6 overexpression dataset and _in vitro_ cultured PCFs, taken from Briggs and colleagues (\cite{briggs2023}).

Appendix file 29 

Table showing the GO terms (calculated using TriTrypDB) derived from the the DE genes which were higher expressed in the early PCFs contained within the RBP6 overexpression dataset compared with \emph{in vitro} PCFs.

Appendix file 30 

Table showing the GO terms (calculated using TriTrypDB) derived from the the DE genes which were higher expressed in the _in vitro_ PCFs compared with early PCFs contained within the RBP6 overexpression dataset.

Appendix file 31 

Table showing the enrichment of metabolic processes (as derived from KEGG and Metacyc) in the DE genes which were higher expressed in the early PCFs contained within the RBP6 overexpression dataset compared with _in vitro_ PCFs.

Appendix file 32 

Table showing the enrichment of metabolic processes (as derived from KEGG and Metacyc) in the DE genes which were higher expressed in the _in vitro_ PCFs compared with the early PCFs contained within the RBP6 overexpression dataset .

